app.js

```diff
@@ -59,3 +59,5 @@ const serverHttp = http
 
 // socket.io 서버 시작 및 app 인스턴스에 저장
 app.set("io", startSocketServer(serverHttp));
+
+require("./src/schedules/index")(app.get("io"));
```

Comment:
`index.js` 는 생략해도 될듯요

```suggestion
require("./src/schedules")(app.get("io"));
```